.travis.yml

```diff
@@ -32,13 +32,21 @@ jobs:
     - make -C enterprise-suite openshift-backend-tests
 
   - name: Backend E2E Go Tests - Minikube
+    cache:
+      directories:
+        - $HOME/.cache/go-build
+        - $HOME/gopath/pkg/mod
```

Comment:
What does this accomplish? If it happens in parallel, I don't think anything is shared.

Reply:
These get cached between consecutive builds. Before running the script the contents in these directories will be restored to the state that was after a previous run. So all the go dependency modules don't have to be downloaded & compiled again.